web/packages/teleport/src/UnifiedResources/FilterPanel.tsx

```diff
@@ -151,6 +152,14 @@ const FilterTypesMenu = ({
     setKinds(newKinds);
   };
 
+  const handleSelectAll = () => {
+    setKinds(kindOptions.map(k => k.value));
+  };
+
+  const handleClearAll = () => {
+    setKinds([]);
+  };
+
```

Comment:
These methods seem a bit simple atm, but when we start to add more filter types besides just `kinds` they will grow a bit.